components/Text Utilities/text_utilities.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Text Utilities</title>
+    <link rel="stylesheet" href="text_utilities.css" />
+  </head>
+  <body>
+    <div class="container-grid">
+      <header class="header">
+        <h1 class="logo"><a href="#">RapidUI</a></h1>
+
+        <div class="header-nav">
+          <nav class="nav">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="#">Documentation</a></li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <aside>
+        <hr>
+        <div class="sidebar">
+          <h4>Components</h2>
+          <ul>
+            <li><a href="/components/Avatar/avatar.html" class="remove-deco">Avatar</a></li>
+            <li><a href="/components/Alert/alert.html" class="remove-deco">Alert</a></li>
+            <li><a href="/components/Badge/badge.html" class="remove-deco">Badge</a></li>
+            <li><a href="/components/Button/button.html" class="remove-deco">Button</a></li>
+            <li><a href="#" class="remove-deco">Card</a></li>
+            <li><a href="/components/Image/image.html" class="remove-deco">Image</a></li>
+            <li><a href="#" class="remove-deco">Input</a></li>
+            <li class="active"><a href="/components/Text Utilities/text_utilities.html" class="active-color remove-deco">Text Utilities</a></li>
+            <li><a href="/components/Navigation/navigation.html" class="remove-deco">Navigation</a></li>
+          </ul>
+        </div>
+      </aside>
+      <main>
+        <h1>Text Utilities</h1>
+        <p class="main-para">
+          Different styles of texts can be viewed below
+        </p>
+        <h1>Headings</h1>
+        <p class="main-para">Below are different types of headings</p>
+        <h1 class="heading1">I am h1 heading</h1>
+        <h2 class="heading2">I am h2 heading</h2>
+        <h3 class="heading3">I am h3 heading</h3>
+        <h4 class="heading4">I am h4 heading</h4>
+        <h5 class="heading5">I am h5 heading</h5>
+        <h6 class="heading6">I am h6 heading</h6>
+
+        <h1>Small Text</h1>
+        <p class="main-para">Below is a small text</p>
+        <small class="text-small">This text will appear in smaller font than normal text.</small>
+        <br><br>
+        <h1>Gray Text</h1>
+        <p class="main-para">Below is a gray text</p>
+        <p class="text-gray">This text will appear in gray color</p>
+        <br><br>
+        <h1>Centre text</h1>
+        <p class="main-para">Below is a centered text</p>
+        <div class="text-centre">This text will appear in the centre</div>
+
+        <iframe
+          src="https://carbon.now.sh/embed/yp5S7ekMYo5ugHrQFfpt"
+          style="width: 680px; height: 540px; border:0; transform: scale(1); overflow:hidden;"

```

Comment:
Inline styling being used, that needs to be avoided

Reply:
Hey so this is actually taken from a external website carbon.now.sh to display code in prettier way. So it was advisable not to change stuff from it, hence i have put it as it is. Rest i have not used inline styling anywhere